### PR TITLE
removed throw and else on container action handler

### PR DESCRIPTION
**Pull request by nikola-jokic**

### CODE REVIEW

**Comment by TingluoHuang** [comment]
Any linked issue for the fix?

**Review by fhammerl — DISMISSED** [review]
LGTM

**Comment by fhammerl** [comment]
Release Notes: Fixed a bug where in a container action, image values like e.g. `dockerfile.foo.bar` or `example/Dockerfile.test` are treated as existing docker images in the registry instead of the filenames they likely are.

File: src/Runner.Worker/Container/DockerUtil.cs [hunk 1]
```diff
@@ -62,6 +63,16 @@ public static string ParseRegistryHostnameFromImageName(string name)
...
         }
 
+        public static bool IsDockerfile(string image)
+        {
+            if (image.StartsWith("docker://", StringComparison.OrdinalIgnoreCase))
+            {
+                return false;
+            }
+            var imageWithoutPath = image.Split('/').Last();
+            return imageWithoutPath.StartsWith("Dockerfile.") || imageWithoutPath.StartsWith("dockerfile.") || imageWithoutPath.EndsWith("Dockerfile") || imageWithoutPath.EndsWith("dockerfile");
```
Comment (TingluoHuang) [inline src/Runner.Worker/Container/DockerUtil.cs:73]:
```suggestion
            return imageWithoutPath.StartsWith("Dockerfile.", StringComparison.OrdinalIgnoreCase) || imageWithoutPath.EndsWith("Dockerfile", StringComparison.OrdinalIgnoreCase);
```

**Review by TingluoHuang — APPROVED** [review]

**Pull request merged**